source/controller/project_controller.py

```diff
@@ -415,21 +439,31 @@ def on_save_graph(self):
 
         file_path = project_information_store.file_path
         file_name, file_type = os.path.splitext(file_path)
+        current_graph = project_information_store.current_graph
         graph = None
         new_g6 = ""
         replaced_line = ""
 
-        if self.edited_graph is None:
+        if current_graph is None:
             return
         else:
-            new_g6 = nx.to_graph6_bytes(self.edited_graph)[10:-1].decode("utf-8")
+            new_g6 = nx.to_graph6_bytes(current_graph, header=False).decode("utf-8")[:-1]
 
         if file_type == ".g6" or file_type == ".txt":
             graph = change_g6_file(file_path, new_g6, current_index)
         if file_type == ".json":
-            graph = change_json_file(file_path, new_g6, current_index)
+            dlg = QMessageBox()
+            dlg.setIcon(QMessageBox.Information)
+            dlg.setText("Graph editing is not allowed in .json format, only in .g6, .txt and .gml formats. "
+                        "You can export this list of graphs to g6 by pressing right button on the desired file.")
```

Comment:
Change the text to: 
"Graph editing is not allowed in .json format, only in .g6, .txt and .gml formats. To allow manipulation and saving: \n"
"You can export the entire list of graphs list of graphs to g6 by pressing right button on the desired file. \n"
"Or you can convert this specific graph with the options available in the toolbar"